src/KeyVault/KeyVault/Commands/Certificate/ImportAzureKeyVaultCertificate.cs

```diff
@@ -106,8 +109,25 @@ public class ImportAzureKeyVaultCertificate : KeyVaultCmdletBase
         [Parameter(Mandatory = false,
                     ParameterSetName = ImportWithPrivateKeyFromStringParameterSet,
                     HelpMessage = "Specifies the password for the certificate and private key base64 encoded string to import.")]
+        [Parameter(Mandatory = false,
+                    ParameterSetName = ImportWithPrivateKeyFromCollectionParameterSet,
+                    HelpMessage = "Specifies the password for the certificate collection and private key to import.")]
```

Comment:
Only first HelpMessage works, please change first HelpMessage to an common description.

Reply:
The original description is common enough.